scripts/exregional_run_analysis.sh

```diff
@@ -867,6 +908,40 @@ fi
 #-----------------------------------------------------------------------
 #
 # comment out for testing
+if [[ ${gsi_type} == "ANALYSIS" && ${ob_type} == "AERO" ]]; then
+
+if [ ${BKTYPE} -eq 1 ]; then
+ print_info_msg "$VERBOSE" "
+ Coldstart skip GSI SD DA"
+else
+# backup background 
+  if [ "${IO_LAYOUT_Y}" == "1" ]; then
+    cp -fr ${bkpath}/fv_tracer.res.tile1.nc ${bkpath}/fv_tracer.res.tile1.nc.org
+  else
+    for ii in ${list_iolayout}
+    do
+      iii=`printf %4.4i $ii`
+      cp -fr  ${bkpath}/fv_tracer.res.tile1.nc.${iii} ${bkpath}/fv_tracer.res.tile1.nc.${iii}.org
+    done
+  fi
+
+  $APRUN ./gsi.x < gsiparm.anl > stdout 2>&1 || print_err_msg_exit "\
+  Call to executable to run GSI returned with nonzero exit code."
+
+# copy updated fv3_tracer back to  restart
+  if [ "${IO_LAYOUT_Y}" == "1" ]; then
+    cp -fr  fv3_tracer ${bkpath}/fv_tracer.res.tile1.nc
+  else
+    for ii in ${list_iolayout}
+    do
+      iii=`printf %4.4i $ii`
+      cp -fr   fv3_tracer.${iii} ${bkpath}/fv_tracer.res.tile1.nc.${iii}
+    done
+  fi
+fi
+
+else
```

Comment:
We don't need line 911-943.  The original fv_tracer files can be found under NWGES

Reply:
@guoqing-noaa 
I will double check. We have two GSI DA runs. The first is the SD DA run, then the meteorological DA fellowed. That is why we need to copy the updated tracer files back before the the meteorological DA. I recalled this is necessary. I will check again. 


To summarize, I will change: 

laeroana_fv3smoke=${laeroana_fv3smoke} 
   i_use_2mQ4B=${i_use_2mq4b},
   i_use_2mT4B=${i_use_2mt4b},

and 
delete the lines line 911-943 IF they are redundant. 

Please let me know if you have any other comment or questions. 

Thanks again for your second review. 

Hongli

Reply:
fv3_tracer is linked from under restart.  So there should be no need to copy back to restart. 

`i_use_2mQ4B=${i_use_2mq4b},`       I think the second q should be a capital Q.

Reply:
@guoqing-noaa 
You are correct. tracer file is linked to a target file, so don't need to be copied. 

I just pushed back my new update. Please have a look. Thank you.